gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/OrganizationLicenseRepository.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.management.api;
+
+import io.gravitee.repository.exceptions.TechnicalException;
+import io.gravitee.repository.management.model.OrganizationLicense;
+import java.util.List;
+import java.util.Set;
+
+public interface OrganizationLicenseRepository extends CrudRepository<OrganizationLicense, String> {
+    List<OrganizationLicense> findAllByOrganizationId(String organizationId) throws TechnicalException;
```

Comment:
There should be only one license per organization so I would return a single result

Reply:
There "should" be, but there is no way to guarantee that the organization id is unique in the table. I was thinking that the service could handle how to only return one of the results.

Reply:
You could make the organization Id the primary key in that table and that garantees that it's unique. WDYT ?